semantics.json

```diff
@@ -159,6 +159,88 @@
                       }
                     ]
                   },
+                  {
+                    "name": "buttonColor",
+                    "type": "text",
+                    "widget": "colorSelector",
+                    "label": "Button Color",
```

Comment:
Do we need translations for this?

Reply:
I can add the fields for them in the language json, but I dont see that the surrounding fields (buttonSize, displayAsButton etc,) gets translated either, when a different language in localhost is applied